project1/css/index.css

```diff
@@ -62,3 +62,113 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+
+.container{
+	width:888px;
+	margin: auto;
+}
+
+header {
+	display: flex;
+	width: 100%;
+	padding: 66px 0;
+}
+
+nav{
+	width:100%;
+	line-height: 44px;
+}
+
+ul {
+	display: flex;
+	justify-content: space-between;	
+}
+
+.logo{
```

Comment:
Something that you're doing to the logo is causing it to squish into a smaller size than what's natural so it looks pixelated. If this were a real company's website you were being paid to develop, they wouldn't like it if their logo got messed up like that, so it matters!